app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -614,11 +614,11 @@
 				TargetAttributes = {
 					510254221C0A60670045FCD8 = {
 						CreatedOnToolsVersion = 7.1;
-						DevelopmentTeam = 4R4XY9R8JR;
+						DevelopmentTeam = GN257VN844;
```

Comment:
`CO CODE -> WE GOTO 12` ( we goto 12 is me + @dbgrandi ATM ) - still waiting on the CP apple account.